odoo/addons/base/models/assetsbundle.py

```diff
@@ -306,6 +312,14 @@ def save_attachment(self, type, content):
 
         self.clean_attachments(type)
 
+        # For end-user assets (common and backend), send a message on the bus
+        # to invite the user to refresh their browser
+        if self.env and 'bus.bus' in self.env and self.name in self.TRACKED_BUNDLES:
+            channel = (self.env.registry.db_name, 'bundle_changed')
+            message = (self.name, self.version)
```

Comment:
message = "reload"

Reply:
We dont manage version we always sendone,  the client queue them for 60 seconds and only display the message after

Reply:
And remove test

Reply:
@antonylesuisse 
This is not for version control, this is to prevent the display of the notification at first asset computation with one user.
Use case:
- assets are not computed yet
- a user logs in
- assets are computed and consequently trigger bus sendones
- page loads in browser
- JS bus fetches messages
In this case, the sendones contain the hash of the assets that are already loaded, there is no need to display the notif

"reload" as message seem too broad

Alright for removing tests, but they are written already..... there is no more work to do to ensure feature's longevity